### DISCUSSION TITLE
Find Some Sample Songs

### DISCUSSION THREAD

**Discussion by mxd1760** [mxd1760/song-typing#30] [Ideas]

**Comment by mxd1760** [comment]
since we want both English and Japanese songs I decided to pick 10 of each to start
English: (from https://www.billboard.com/charts/greatest-hot-100-singles/)
 1- <https://music.youtube.com/watch?v=J7p4bzqLvCw>
 2- <https://music.youtube.com/watch?v=vuQyUMOK-K0>
 3- <https://music.youtube.com/watch?v=mom2TLs3Fi0>
 4- <https://music.youtube.com/watch?v=ygVgxGSQIsw>
 5- <https://music.youtube.com/watch?v=tYvFa2ARD24>
 6- <https://music.youtube.com/watch?v=LAJ9GoMwXTs>
 7- <https://music.youtube.com/watch?v=wO2TLZ6Pqo4>
 8- <https://music.youtube.com/watch?v=OIPmhkzN2ug>
 9- <https://music.youtube.com/watch?v=ulMRK4grORk>
10- <https://music.youtube.com/watch?v=xTvyyoF_LZY>
Japanese: (from https://open.spotify.com/playlist/37i9dQZEVXbKXQ4mDTEBXq)
 1- <https://music.youtube.com/watch?v=DO_aopUeFnw>
 2- <https://music.youtube.com/watch?v=MT0xgNcOnsA>
 3- <https://music.youtube.com/watch?v=wsKw_Astidg>
 4- <https://music.youtube.com/watch?v=_GXbXU4g_CA>
 5- <https://music.youtube.com/watch?v=hID1Wt1ByEQ>
 6- <https://music.youtube.com/watch?v=s4ZThuWQdhA>
 7- <https://music.youtube.com/watch?v=m-Fhzepy7L8>
 8- <https://music.youtube.com/watch?v=Jg73p9udbAQ>
 9- <https://music.youtube.com/watch?v=-QxMzUEJH4Q>
10-<https://music.youtube.com/watch?v=7Ze7MDQjXS0>
Extras:
 <https://music.youtube.com/watch?v=sEcvQvftOQY> - This is a Japanese top 10 song that's just all English lyrics

Copyright might be a problem but if we can make sure that the app accesses the songs from user-submitted links and then processes them into usable files instead of coming with saved files we should be fine because we wouldn't be distributing the songs. also, one of the app's goals was to have a good way to take user files/links and help them get things set up.

**Comment by Jayman2000** [comment]
I think that it would be funny to have levels for Marisa Stole the Precious Thing and Chirumiru Cirno.
